<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_setFont">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fontSizeUp">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_printTextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clearTextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Text Edit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setFontItalic">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Italic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setFontColor">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Color Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fontSizeDown">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
